hr_attendance_validation/models/hr_attendance.py

```diff
@@ -0,0 +1,94 @@
+# Copyright 2021 Pierre Verkest
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+from odoo import SUPERUSER_ID, _, api, fields, models
+from odoo.exceptions import ValidationError
+
+
+class HrAttendance(models.Model):
+    _inherit = "hr.attendance"
+
+    is_overtime_due = fields.Boolean(
+        string="Is overtime due",
+        default=False,
+        help="What ever the overtime is due or not. "
+        "By default overtime are not due until manager validate them.",
```

Comment:
```suggestion
        help="Whether the overtime is due or not. "
        "By default overtime is not due until a manager validates it.",
```
There may be a better word than "due" to make the concept clearer, but I can't think of it...

Reply:
does "owed" is proper than "due" ?